config/routes.rb

```diff
@@ -166,14 +166,16 @@
       get   '/races/:race', to: 'races#show',   as: 'race'
       patch '/races/:race', to: 'races#update'
 
-      get    '/races/:race/entry', to: 'races/entries#show'
-      put    '/races/:race/entry', to: 'races/entries#create'
-      patch  '/races/:race/entry', to: 'races/entries#update'
-      delete '/races/:race/entry', to: 'races/entries#destroy'
+      get    '/races/:race/entries/:id', to: 'races/entries#show',   as: 'race_entry'
```

Comment:
How come we're switching back to ID based?

Reply:
Since a ghost is an Entry, that means one user can own multiple entries. So consumers need some way of differentiating one from another, which means we can't do `/races/:race/entry` anymore.